lib/node_modules/@stdlib/stats/base/dists/cauchy/cdf/src/main.c

```diff
@@ -0,0 +1,47 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/stats/base/dists/cauchy/cdf.h"
+#include "stdlib/math/base/assert/is_nan.h"
+#include "stdlib/math/base/special/atan2.h"
+
+const double ONE_OVER_PI = 0.3183098861837907;
```

Comment:
```suggestion
static const double ONE_OVER_PI = 0.3183098861837907;
```

This will limit the scope of the variable to the file in which it is declared, making it invisible to other files.